Classes/common/CDTReplicator/CDTAbstractReplication.m

```diff
@@ -123,6 +167,7 @@ - (NSDictionary *)dictionaryForReplicatorDocument:(NSError *__autoreleasing *)er
 
         doc[@"headers"] = self.optionalHeaders;
     }
+    doc[@"interceptors"] = self.httpInterceptors;
```

Comment:
Yes :disappointed: it's so we can pass the interceptors to the creation method for the TDReplicator